solidity/contracts/bridge/Bridge.sol

```diff
@@ -123,6 +123,12 @@ contract Bridge is Governable, EcdsaWalletOwner {
 
     event MovedFundsSwept(bytes20 walletPubKeyHash, bytes32 sweepTxHash);
 
+    event MovedFundsSweepRequestTimedOut(
```

Comment:
We have `RedemptionTimedOut` so maybe `MovedFundsSweepTimedOut`?

Reply:
I think `MovedFundsSweepTimedOut` makes sense especially given the parameter names:
https://github.com/keep-network/tbtc-v2/blob/b06948ece5083d659bbcd3782bda531959e750e4/solidity/contracts/bridge/Bridge.sol#L193-L195